### ISSUE TITLE
Have questions?

### ISSUE THREAD

**Issue by thetutlage** [adonisjs/adonis-websocket-protocol#1]
Read this twitter thread before you tell me that I am wrong. https://twitter.com/adonisframework/status/965843000742981632

I did my research and please you do yours, otherwise....

**Comment by molobala** [comment]
Broken depedencies listed in readme.md , simple-filereader and  @adonisjs/websocket-packets can not be found on npm !

**Comment by thetutlage** [comment]
Yes, this is a paper, packages will be published soon

**Comment by molobala** [comment]
Ah okay I see! 
Thank you for you reply !
I guess you will add websocket functionnality to adonis next release , I'm working on an adonis project , and will need to make use of websocket, so it will be amzing to include it in the next version of adonis

**Comment by ntvsx193** [comment]
Any info for published packages?

**Comment by thetutlage** [comment]
Yes all have been published apart from `@adonisjs/websocket-client`, which will be out soon
